### ISSUE TITLE
No "compile / upload" button on windows

### ISSUE THREAD

**Issue by mrninhvn** [DIYOSEPP/oseppBlock#2]
![image](https://user-images.githubusercontent.com/30751439/83262558-64dddb80-a1e7-11ea-9c2e-d1d77df87c30.png)

Rebuild Complete but block doesn't display.

**Comment by zhsvector** [comment]
Hello, There is no "compile / upload" button, which means that the nodejs module failed to load.
Please confirm whether you executed ./node_modules/.bin/electron-rebuild -f
To compile the nodejs module for electron
I assume that you use the windows system, then you can open the Developer Tools through the shortcut key [ ctrl+ shift + i ] to further see what is wrong.
(or press the [alt] key, a menu will appear at the top, select View-> Toggle Developer Tools)
In addition, if you just want to use the software, download a pre-compiled version is a better choice.

Best Regards,
OSEPP team

**Comment by mrninhvn** [comment]
Thank you @zhsvector 
![image](https://user-images.githubusercontent.com/30751439/83287383-c57f0f80-a20b-11ea-8d71-301d20485e2b.png)
Rebuild OK for me.
![image](https://user-images.githubusercontent.com/30751439/83287434-dcbdfd00-a20b-11ea-99b9-c574cbb064ad.png)
This is my error, can you hekp me, thank you again.

**Comment by mrninhvn** [comment]
@zhsvector 
I use Closure Library v20200112
![image](https://user-images.githubusercontent.com/30751439/83288732-109a2200-a20e-11ea-88df-666e4d969df6.png)

**Comment by zhsvector** [comment]
Yes, this is caused by the Closure Library. They moved this function out of the closure-library.
https://github.com/google/closure-library/commit/9325ce1bbebcfe8c7aa54f8342959c73a1762a77
So we need to remove it from the scratch-block, or use the older closure-library.

**Comment by mrninhvn** [comment]
@zhsvector Thank you for your support.
![image](https://user-images.githubusercontent.com/30751439/83320765-ae283c80-a274-11ea-809c-63615c377749.png)
There is no "compile / upload" button, what version of Closure Library are you using?

**Comment by zhsvector** [comment]
I repeated the rebuild steps in a brand new linux system, using closure-library-20191027.  This problem is not reproduced.
I suggest you set a breakpoint and trace on line 293 of serial.js to further find out what went wrong.
another suggestion is to delete the node_modules directory and execute npm install && ./node_modules/.bin/electron-rebuild -f again
